src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -109,34 +196,54 @@ def update_data(self):
             redis_client = self.db_conn.get_redis_client(self.db_conn.APPL_DB)
             db = self.db_conn.db_map[self.db_conn.APPL_DB]["db"]
             self.pubsub = redis_client.pubsub()
-            self.pubsub.psubscribe("__keyspace@{}__:{}".format(db, "LLDP_ENTRY_TABLE:*"))
+            self.pubsub.psubscribe("__keyspace@{}__:{}".format(db, mibs.lldp_entry_table(b'Ethernet*')))
```

Comment:
>Ethernet [](start = 91, length = 8)

You add assumption of port name. I think original one is more accurate.

Reply:
I added it to avoid spam of log message about 'eth0' not being a valid interface name. In the end, i just changed dbsyncd to not submit LLDP_ENTRY:eth0 to DB, as it is never used anyway.

Reply:
I see. It's better to check 'not eth0'.

---
In reply to: [205791584](https://github.com/Azure/sonic-snmpagent/pull/72#discussion_r205791584) [](ancestors = 205791584)